ruby/vampires.rb

```diff
@@ -0,0 +1,64 @@
+puts "How many employees will be processed?"
+employees = gets.chomp.to_i
+
+until employees == 0
+
+	puts "What is your name?"
+	name = gets.chomp
+
+	puts "How old are you?"
+	age = gets.chomp.to_i
+
+	puts "What year were you born?"
+	year_born = gets.chomp.to_i
+
+	puts "Our company cafeteria serves garlic bread. Should we order some for you (y/n)?"
+	garlic_bread = gets.chomp
+
+	puts "Would you like to enroll in the company's health insurance (y/n)?"
+	insurance = gets.chomp
+
+	puts "What allergies do you have (type 'done' when finished)?"
+	allergies = gets.chomp
+
+	real_age = 2017 - year_born
+
+	if age == real_age
+		age_is_correct = true
+	else
+		age_is_correct = false
+	end
+
+	allergies_done = false
+
+	until allergies_done 
+		if allergies == "sunshine"
+			allergies_done = true
+		elsif allergies == "done"
+			allergies_done = true
+		else 
+			allergies = gets.chomp
+		end
+	end
+
+	if allergies == "sunshine"
```

Comment:
'Match the following conditions in the order they're listed.' We first read this when starting off the challenge and we may try to hack it out with a basic if/elsif structure but we quickly learn that the first condition to pass will be the result even if a 'more true' version is below.  We may be tempted to switch the order for a working solution, but if we are to follow the constraints we need to be more creative. What if we made each individual scenario it's own respective if block? What would go inside each if block? Surely we can't just puts out the result otherwise our screen would be full of each vampire check that passed. What we could do is keep setting a variable result within each if block and then at the end of all the blocks display the result to look something like this
```
if condition_1
	result = 'low'
end

if condition_2
	result = 'medium'
end

if condition_3
	result = 'high'
end

puts result
```